src/main/java/StringOperations.java

```diff
@@ -0,0 +1,93 @@
+public class StringOperations {
+
+    /*
+    Params : input
+     */
+    public static int getFirstSpace(String input) throws DukeException {
+        int index = input.indexOf(" ");
+        if (index == -1) {
+            throw new DukeException(DukeException.ExceptionResponse.EXCEPTION_NO_DESCRIPTION);
+        }
+        return index;
+    }
+
+    /*
+    Function to print response when new tasks are added to the list
+     */
+    public static void printAddResponse(Task task, int index) {
+        System.out.println("\tGot it. I've added this task:");
+        System.out.println("\t" + task.toString());
+        System.out.println("\tNow you have " + index + " tasks in the list");
+    }
+
+    /*
+    Prints all tasks in the list with the letter heading [D],[E] and status of completion.
+     */
+    public static void printList(Task[] tasks, int taskIndex) {
+        System.out.println("\tHere are the tasks in your list:");
+        for (int i = 0; i < taskIndex; i++) {
+            System.out.println("\t" + (i + 1) + ". " + tasks[i].toString());
+        }
+    }
+
+    /*
+    Function to print the response when a task is marked as completed by the user.
+     */
+    public static void completeTask(Task[] tasks, int taskIndex) {
+        System.out.println("\tNice! I've marked this task as done: ");
```

Comment:
You could consider declaring the constant String as a constant variable in the class instead